ftpvl/fetchers.py

```diff
@@ -68,16 +74,88 @@ def __init__(
         project: str,
         jobset: str,
         eval_num: int = 0,
+        absolute_eval_num: bool = False,
         mapping: dict = None,
         hydra_clock_names: list = None
     ) -> None:
         super().__init__()
         self.project = project
         self.jobset = jobset
         self.eval_num = eval_num
+        self.absolute_eval_num = absolute_eval_num
         self.mapping = mapping
         self.hydra_clock_names = hydra_clock_names
 
+    def _get_builds(self, eval_num: int, absolute_eval_num: bool, params: str = "") -> List[int]:
```

Comment:
If you already have `self.absolute_eval_num` set and equal to the `absolute_eval_num` being passed in here, then skip this parameter and use `self.absolute_eval_num` throughout the body of this function.